deploy_cluster.yml

```diff
@@ -6,6 +6,8 @@
 
 - import_playbook: playbooks/generate_discovery_iso.yml
 
+- import_playbook: playbooks/mount_discovery_iso_for_pxe.yml
```

Comment:
Please a when here to make it clear when this playbook gets run.

Reply:
The when is inside the playbook
`when: MOUNT_DISCOVERY_ISO == True`